content/sql/concepts/dates/terms/between/between.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'BETWEEN()'
+Description: 'Returns records where a specified value is within a given range, including the boundary values.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Date'
+  - 'Queries'
+  - 'SQL Server'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The **`BETWEEN()`** operator is used to find values within a given range. The values can be numbers, texts, or dates.
+
+The **`BETWEEN()`** operator is _inclusive_. Both begin and end values are included.
+
+## Syntax
+
+```sql
+SELECT column_name(s) from table_name where column_name between value1 and value2;
+```
```

Comment:
Let's use `pseudo` instead of `sql` since the code block is representing a syntax and not an example:

````suggestion
```pseudo
SELECT column_name(s) from table_name where column_name between value1 and value2;
```
````